MarvelApp/MarvelApp/Networking/API.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  API.swift
+//  MarvelApp
+//
+//  Created by Lê Tiến Đạt on 12/04/2022.
+//
+
+import Foundation
+
+class APICaller {
```

Comment:
```suggestion
class API {
```
The name of Class should be the same as the name of file that contains it